GettingStarted/Resources.md

```diff
@@ -57,3 +62,10 @@ resources, feel free to [make a pull request](../CONTRIBUTING.md).
   of the above, but it can also be run from the command line.
 * If you want to learn vim, or just use a vim mode in the editor or IDE of your
   choice, you can't go wrong with this [vim cheat sheet](https://vim.rtorr.com).
+
+## General Learning
```

Comment:
while youre at it, could you add Sourcegraph here as well? https://sourcegraph.com/start

Reply:
I'll add it, but it doesn't really seem like "general learning," does it?

Reply:
Hm, `Browser Extensions`?

Reply:
Hmm, unless we rename "repo management" to "useful tools" or something similar?

Reply:
well this doesnt have anything to do with repo management, i reckon sourcegraph should be in a separate category - I think `Browser Extensions` would be good, since I feel there are probably some other useful ones ([this](https://chrome.google.com/webstore/detail/refined-github/hlepfoohegkhhmjieoechaddaejaokhf?hl=en) comes to mind, as well as the [official github extentions list](https://github.com/collections/github-browser-extensions))

Reply:
Yeah, but what I mean is that we could create a broader category for all the “useful tools” like Pull Reminders, Sourcegraph, etc. Then repo management could probably be subsumed by the broader category (or kept separate).

Reply:
I think more categories is better than too few categories, for ease of browsing 👓

Reply:
for the uninitiated it's also easier to understand: "useful tools" could mean anything, and it becomes a bunch of stuff that doesnt really stand out individually

Reply:
Okay fair enough. That should probably be another pull request then - seems like you have quite a few resources at hand 😛